scipy/integrate/_quadpack_py.py

```diff
@@ -330,6 +334,29 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
         Clenshaw-Curtis integration is used on those intervals containing the
         point :math:`x = c`.
 
+    ** Integration of Complex Function of a Real Variable **
+
+    This method integrates the real and complex components separately.
+    Doing so makes the further assumption that the integrals of these
+    components exist over the given interval.
+
+    A complex valued function, :math:`f`, of a real variable can be written as
+    :math:`f = g + ih`.  Similarly, the integral of :math:`f` can be
+    written as
+
+    .. math::
+        \\int_a^b f(x) dx = \\int_a^b g(x) dx + i\\int_a^b h(x) dx
+
+    This assumes that the integrals of :math:`g` and :math:`h` exist
+    over the inteval :math:`[a,b]` [2]_.
+
+    For example,
+
+    .. math::
+        \\int_0^{\\frac{\\pi}{2}} e^{it} dt =
+        \\int_0^{\\frac{\\pi}{2}} \\cos{t} + i\\sin{t} dt
```

Comment:
I don't think this is needed here. If you'd like to have an example, let's put it in the code examples below.

Reply:
I went ahead and removed this.  I agree, it's not necessary.